code/modules/client/preferences.dm

```diff
@@ -1390,11 +1398,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 					if(new_hair)
 						active_character.hair_color = sanitize_hexcolor(new_hair)
 				if("hair_style")
-					var/new_hair_style
-					if(active_character.gender == MALE)
-						new_hair_style = input(user, "Choose your character's hair style:", "Character Preference")  as null|anything in GLOB.hair_styles_male_list
-					else
-						new_hair_style = input(user, "Choose your character's hair style:", "Character Preference")  as null|anything in GLOB.hair_styles_female_list
+					var/new_hair_style = input(user, "Choose your character's hair style:", "Character Preference")  as null|anything in GLOB.hair_styles_list
```

Comment:
Not super sure how I feel about the global hair style lists without any kind of sorting. I find it hard to find hair styles that fit characters at the moment on the mirror, which I believe can use all. It would be nice if there was some categorisation so finding appropriate hair styles for what you want is easier.

Reply:
Maybe a button that lets you change between female and male lists?

Reply:
This is not a problem of separation, it's that there's no icon previews. Leave it, it's out of scope imo

Reply:
I don't think its out of scope considering this PR is changing the behaviour to make it arguably worse to sort through

Reply:
Could make it be something similar to how I did the mirror choice. 

A choice between picking from the Male or Female Hairstyles, although don’t think that wouldn’t exactly fix the problem?

Reply:
You could change it to use tgui_input_list which is at least searchable

Reply:
really should have atomised the gender neutrality part out and kept this bit seperate